### PR TITLE
Added pypi command

### PR DESCRIPTION
**Pull request by kandasamyc**
Fixes issue #653

### CODE REVIEW

**Review by pedrofracassi — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution. Could you please change `let` to `const` where changing the variable isn't necessary?

**Comment by SamC302** [comment]
Will do! Once question, does the test fail because of code style? Or is there something else I need to fix?

**Comment by Doges** [comment]
> 
> 
> Will do! Once question, does the test fail because of code style? Or is there something else I need to fix?

Yeah, its the code style.

File: package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "lodash": "^4.17.19",
     "mal-scraper": "^2.5.0",
     "mathjs": "^5.7.0",
-    "moment": "^2.24.0",
+    "moment": "^2.29.1",
```
Comment (Doges) [inline package.json:65]: it really needs a newer version?

Comment (SamC302) [reply package.json:65]: Nope, this is my bad. I think I might have done this on accident trying to fix something. I'll revert it.

File: src/apis/PyPI.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    const data = {}
+    const parsed = cheerio.load(html.data)
+    const title = parsed('.package-header__name').text().trim()
+    const desc = parsed('.package-description__summary').text().trim()
+    const pipCommand = parsed('#pip-command').text().trim()
+    const date = parsed('.package-header__date').text().trim()
+    parsed('.vertical-tabs__tab').each(
+      function (i, e) {
+        if (parsed(this).text().trim() === 'Homepage') {
+          data['package_link'] = parsed(this).attr('href')
```
Comment (fernando457829) [inline src/apis/PyPI.js:43]:
```suggestion
          data.packageLink = parsed(this).attr('href')
```

File: src/apis/PyPI.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        }
+      }
+    )
+    data['description'] = desc
+    data['name'] = title
+    data['command'] = pipCommand
+    data['date'] = date
```
Comment (fernando457829) [inline src/apis/PyPI.js:50]:
```suggestion
    data.description = desc
    data.name = title
    data.command = pipCommand
    data.date = date
```

**Review by fernando457829 — CHANGES_REQUESTED** [review]

**Comment by almeidx** [comment]
@SamC302 ^ and fix the conflicts

**Comment by SamC302** [comment]
Is there anything else I need to fix?

**Review by fernando457829 — APPROVED** [review]

File: src/apis/PyPI.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    super({
+      name: 'pypiapi'
+    })
+  }
+
+  async search (query) {
+    const html = await axios({
+      url: `${API_URL}search/?q=${query}`
+    })
+    const parsed = cheerio.load(html.data)
```
Comment (almeidx) [inline src/apis/PyPI.js:18]: I feel like having the scraper directly in the bot is not a very good idea. I believe it should stay on its own, like the MDN documentation https://github.com/SwitchbladeBot/mdn-doc using Cloudflare workers

**Pull request closed without merging**